src/shortcuts/redo.ts

```diff
@@ -8,7 +8,15 @@ import { getLatestActionType } from '../util/getLastActionType'
 const redoShortcut: Shortcut = {
   id: 'redo',
   label: 'Redo',
-  description: 'Redo',
+  description: getState => {
+    const lastActionType = getLatestActionType(getState().patches)
```

Comment:
I was mistaken about `getLatestActionType` only working for `undo`! I thought that `patches` were in reverse order, but I tested it and this works great.